src/duckling/project/map-select.component.ts

```diff
@@ -11,14 +11,15 @@ import {openDialog} from '../util/md-dialog';
  */
 @Component({
     selector: "dk-map-select",
-    styleUrls: ["./duckling/layout.css"],
+    styleUrls: ["./duckling/layout.css", "./duckling/project/map-select.component.css"],
     template: `
         <div *ngIf="!listLoaded">
             <md-spinner></md-spinner>
         </div>
         <div *ngIf="listLoaded">
             <dk-section headerText="Select an Existing Map">
-                <md-nav-list>
+                <md-nav-list
+                    class="existing-map-list">
```

Comment:
We usually wouldn't add a class like this and just style the md-nav-list itself since our components should stay small and styling something like a "md-nav-list" is already identifying.